### ISSUE TITLE
Game crashed when i tried place a jug with meal/

### ISSUE THREAD

**Issue by dann1kid** [anegostudios/VintageStory-Issues#1420]
**Game Version:** 1.15.9  
**Platform:** Windows 
**Modded:**  Yes 
**SP/MP:**  Multiplayer 

### Description
Place jug on table with meal throw error

### How to reproduce
1. Make a 'meal'
2. Use it on 'place on table'
3. It crashes the game

### Expected behavior
Place object or error message on screen to unable to place jug

### Logs
```
Running on 64 bit Windows with 96 GB RAM 
Version: v1.15.9 (Stable)20.11.2021 21:44:18: Critical error occurred
System.InvalidOperationException: OnBlockPlaced stack copy failed. Trying to set 7 stacks on an inventory with 6 slots
   в Vintagestory.GameContent.BlockEntityContainer.OnBlockPlaced(ItemStack byItemStack)
   в Vintagestory.GameContent.BlockEntityCrock.OnBlockPlaced(ItemStack byItemStack)
   в Vintagestory.Client.NoObf.ClientWorldMap.SpawnBlockEntity(String classname, BlockPos position, ItemStack byItemStack)
   в Vintagestory.API.Common.Block.OnBlockPlaced(IWorldAccessor world, BlockPos blockPos, ItemStack byItemStack)
   в Vintagestory.Common.BlockAccessorRelaxed.SetBlock(Int32 blockId, BlockPos pos, ItemStack byItemstack)
   в Vintagestory.API.Common.Block.DoPlaceBlock(IWorldAccessor world, IPlayer byPlayer, BlockSelection blockSel, ItemStack byItemStack)
   в Vintagestory.GameContent.BlockBehaviorHorizontalOrientable.TryPlaceBlock(IWorldAccessor world, IPlayer byPlayer, ItemStack itemstack, BlockSelection blockSel, EnumHandling& handling, String& failureCode)
   в Vintagestory.API.Common.Block.TryPlaceBlock(IWorldAccessor world, IPlayer byPlayer, ItemStack itemstack, BlockSelection blockSel, String& failureCode)
   в Vintagestory.Client.NoObf.ClientMain.OnPlayerTryPlace(BlockSelection blockSelection, String& failureCode)
   в Vintagestory.Client.NoObf.SystemMouseInWorldInteractions.OnBlockBuild(BlockSelection blockSelection, Block onBlock, String& failureCode)
   в Vintagestory.Client.NoObf.SystemMouseInWorldInteractions.HandleMouseInteractionsBlockSelected(Single dt)
   в Vintagestory.Client.NoObf.SystemMouseInWorldInteractions.OnFinalizeFrame(Single dt)
   в Vintagestory.Client.NoObf.ClientEventManager.TriggerRenderStage(EnumRenderStage stage, Single dt)
   в Vintagestory.Client.NoObf.ClientMain.TriggerRenderStage(EnumRenderStage stage, Single dt)
   в Vintagestory.Client.NoObf.ClientMain.RenderToDefaultFramebuffer(Single dt)
   в _rnlVSzJ2AGPu2719D3re8Z7kN2c._TshCCJvNc6laaisiRTK6RNaXWQ1A(Single )
   в _o0lbNJXQO06me4d8j661Kh7y0zO._HmSBpw6fP4XFLEcv29tc0ifOKYvb(Single )
   в _o0lbNJXQO06me4d8j661Kh7y0zO._3U99eoPu0PaDLPVrL8tfzH5UDEJ(Single )
   в Vintagestory.Client.NoObf.ClientPlatformWindows.window_RenderFrame(Object sender, FrameEventArgs e)
   в System.EventHandler`1.Invoke(Object sender, TEventArgs e)
   в OpenTK.GameWindow.RaiseRenderFrame(Double elapsed, Double& timestamp)
   в OpenTK.GameWindow.DispatchRenderFrame()
   в OpenTK.GameWindow.Run(Double updates_per_second, Double frames_per_second)
   в _M48MnHH5BAvHUzcYoEuchRW0acb._e0atQ8VmmvgBK7zb3Yh4iDqOiHY(_UGHNIsOGTJBe1BTRXYmLwHTbmS , String[] )
   в _xNqQDGqRsRSMIeuzt2MHKQHDzSd._e0atQ8VmmvgBK7zb3Yh4iDqOiHY(ThreadStart )
-------------------------------

Event Log entries containing Vintagestory.exe, the latest 3
==================================
{ TimeGenerated = 14.11.2021 1:36:14, Site = , Source = Application Error, Message = Имя сбойного приложения: Vintagestory.exe, версия: 1.15.9.0, метка времени: 0x6187ffbc
Имя сбойного модуля: KERNELBASE.dll, версия: 10.0.19041.1320, метка времени: 0x00d7bca2
Код исключения: 0xc0020001
Смещение ошибки: 0x0000000000034f69
Идентификатор сбойного процесса: 0x51a8
Время запуска сбойного приложения: 0x01d7d8de2176f6d0
Путь сбойного приложения: C:\Vintagestory159\Vintagestory.exe
Путь сбойного модуля: C:\Windows\System32\KERNELBASE.dll
Идентификатор отчета: 518011f2-d4f3-4ced-b124-b38069dedd33
Полное имя сбойного пакета: 
Код приложения, связанного со сбойным пакетом:  }
--------------
{ TimeGenerated = 14.11.2021 1:36:13, Site = , Source = .NET Runtime, Message = Приложение: Vintagestory.exe
Версия платформы: v4.0.30319
Описание. Процесс был завершен из-за необработанного исключения.
Сведения об исключении: код исключения c0020001, адрес исключения 00007FF8CB3D4F69
Стек:
   в System.Environment._Exit(Int32)
   в System.Environment._Exit(Int32)
   в System.Environment.Exit(Int32)
   в System.EventHandler`1[[System.__Canon, mscorlib, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089]].Invoke(System.Object, System.__Canon)
   в OpenTK.NativeWindow.OnClosedInternal(System.Object, System.EventArgs)
   в System.EventHandler`1[[System.__Canon, mscorlib, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089]].Invoke(System.Object, System.__Canon)
   в OpenTK.Platform.Windows.WinGLNative.HandleDestroy(IntPtr, OpenTK.Platform.Windows.WindowMessage, IntPtr, IntPtr)
   в OpenTK.Platform.Windows.WinGLNative.WindowProcedure(IntPtr, OpenTK.Platform.Windows.WindowMessage, IntPtr, IntPtr)
   в OpenTK.Platform.Windows.Functions.DestroyWindow(IntPtr)
   в OpenTK.Platform.Windows.Functions.DestroyWindow(IntPtr)
   в OpenTK.Platform.Windows.WinGLNative.DestroyWindow()
   в OpenTK.Platform.Windows.WinGLNative.WindowProcedure(IntPtr, OpenTK.Platform.Windows.WindowMessage, IntPtr, IntPtr)
   в OpenTK.Platform.Windows.Functions.DefWindowProc(IntPtr, OpenTK.Platform.Windows.WindowMessage, IntPtr, IntPtr)
   в OpenTK.Platform.Windows.Functions.DefWindowProc(IntPtr, OpenTK.Platform.Windows.WindowMessage, IntPtr, IntPtr)
   в OpenTK.Platform.Windows.WinGLNative.WindowProcedure(IntPtr, OpenTK.Platform.Windows.WindowMessage, IntPtr, IntPtr)
   в OpenTK.Platform.Windows.Functions.PeekMessage(OpenTK.Platform.Windows.MSG ByRef, IntPtr, Int32, Int32, OpenTK.Platform.Windows.PeekMessageFlags)
   в OpenTK.Platform.Windows.Functions.PeekMessage(OpenTK.Platform.Windows.MSG ByRef, IntPtr, Int32, Int32, OpenTK.Platform.Windows.PeekMessageFlags)
   в OpenTK.Platform.Windows.WinGLNative.ProcessEvents()
   в OpenTK.GameWindow.Run(Double, Double)
   в _M48MnHH5BAvHUzcYoEuchRW0acb._e0atQ8VmmvgBK7zb3Yh4iDqOiHY(_UGHNIsOGTJBe1BTRXYmLwHTbmS, System.String[])
   в _xNqQDGqRsRSMIeuzt2MHKQHDzSd._e0atQ8VmmvgBK7zb3Yh4iDqOiHY(System.Threading.ThreadStart)
   в _M48MnHH5BAvHUzcYoEuchRW0acb._Fvqf1k3vBXhdBOr0rRgzEoDCZQC(System.String[])
 }
--------------
{ TimeGenerated = 24.10.2021 22:56:29, Site = , Source = Application Error, Message = Имя сбойного приложения: Vintagestory.exe, версия: 1.15.6.0, метка времени: 0x614dc2b6
Имя сбойного модуля: OPENGL32.DLL, версия: 10.0.19041.1081, метка времени: 0xc0583cc8
Код исключения: 0xc0000005
Смещение ошибки: 0x000000000001cf3c
Идентификатор сбойного процесса: 0x5498
Время запуска сбойного приложения: 0x01d7c910f2eff5aa
Путь сбойного приложения: C:\Vintagestory156\Vintagestory.exe
Путь сбойного модуля: C:\Windows\SYSTEM32\OPENGL32.DLL
Идентификатор отчета: 5ceeb795-1157-4dc5-823b-c161d697c1eb
Полное имя сбойного пакета: 
Код приложения, связанного со сбойным пакетом:  }

**Comment by Craluminum2413** [comment]
I think it is fixed in 1.16 due to ground storage rework

**Comment by copygirl** [comment]
Very likely. Will assume fixed for now.
